src/ralph/lib/mixins/models.py

```diff
@@ -1,10 +1,47 @@
 # -*- coding: utf-8 -*-
+from django.contrib.contenttypes.models import ContentType
 from django.core.urlresolvers import reverse
 from django.db import models
 from django.utils import timezone
 from django.utils.translation import ugettext_lazy as _
 
 
+def get_admin_url(obj, action):
```

Comment:
these functions shouldn't be in models module, but in <choose one of forbidden names, ex. utils, common etc> :)